frontend-react/src/components/header/DAPHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { useLocation } from "react-router-dom";
+
+const urlsForDAP = [
+    "/login",
+    "/getting-started/",
+    "/how-it-works/",
+    "/terms-of-service",
+];
+
+export const useDAP = (env: string | undefined) => {
```

Comment:
We should account for the possibility of an empty string. While it'll produce `false` since it's not prod, since it's pulling from an env file, letting the dev know through a console warn or error could be nice!

Reply:
You're possibly going to get `undefined` from `process.env...` if the variable doesn't exist. You can either account for that in the condition on 25 here, or in App.tsx on line 96 where it's called. Other than that, looks great!

Reply:
Right! Good call. I expanded on line 25-26 to check/warn for undefined as well